### ISSUE TITLE
libfreeipmi.so rebases in RHEL/Centos7.5

### ISSUE THREAD

**Issue by dcsouthwick** [uwcms/sysmgr#2]
sysmgr is incompatible with RHEL/Centos 7.5 due to rebase to libfreeipmi.so.17

```
Error: Package: sysmgr-1.1.7-1.el7.cern.x86_64 (@uwcms)
           Requires: libfreeipmi.so.13()(64bit)
           Removing: freeipmi-1.2.9-8.el7.x86_64 (@base)
               libfreeipmi.so.13()(64bit)
           Updated By: freeipmi-1.5.7-2.el7.x86_64 (base)
              ~libfreeipmi.so.17()(64bit)
           Available: freeipmi-1.2.9-7.el7.x86_64 (updates)
               libfreeipmi.so.13()(64bit)
```

looks like rebuild required

**Comment by jtikalsky** [comment]
I use the CC7 docker image at https://hub.docker.com/u/cern/ to produce the CC7 builds.  Is there a CC7.5 docker image I can pull from somewhere that you're aware of?  I would be happy to produce another build if I can get a proper environment for it.

**Comment by dcsouthwick** [comment]
I believe the cc7 image you linked is already at 7.5, at least according to https://gitlab.cern.ch/linuxsupport/cc7-base.
```
$ docker pull cern/cc7-base
$ docker run -it cern/cc7-base bash
$ cat /etc/redhat-release
CentOS Linux release 7.5.1804 (Core)
$ yum install sysmgr
Error: Package: sysmgr-1.1.7-1.el7.cern.x86_64 (uwcms)
           Requires: libfreeipmi.so.13()(64bit)
           Available: freeipmi-1.2.9-7.el7.x86_64 (updates)
               libfreeipmi.so.13()(64bit)
           Available: freeipmi-1.2.9-8.el7.x86_64 (updates)
               libfreeipmi.so.13()(64bit)
           Installed: freeipmi-1.5.7-2.el7.x86_64 (@base)
              ~libfreeipmi.so.17()(64bit)
```

**Comment by jtikalsky** [comment]
Released [v1.1.7.1](https://github.com/uwcms/sysmgr/releases/tag/v1.1.7.1) with support for CC7.5.
